utils/Merkelizer.js

```diff
@@ -32,6 +16,81 @@ module.exports = class Merkelizer extends AbstractMerkleTree {
     return res;
   }
 
+  /**
```

Comment:
Next partial PR I would say. We can extend `AbstractMerkleTree` and get a own class for a general 'word-sized'-merkle tree. So we can re-use this for `calldata` and so on 🙂